projects/arlas-toolkit/src/lib/components/search/search-dialog.component.scss

```diff
@@ -67,37 +67,37 @@ $search-width: 400px;
     .autocomplete {
         margin: 4px 2px;
         padding: 0;
-    
+
         li:last {
             padding-bottom: 0;
         }
 
         .search-warning {
             @include search-text;
         }
-    
+
         .search-option {
             @include search-text;
             display: flex;
             justify-content: space-between;
             cursor: pointer;
-    
+
             &:hover {
                 background-color: rgba(208, 217, 226, 0.548);
             }
-    
+
             span {
                 padding: 0 4px;
             }
-    
+
             .count {
                 color: #5a5555;
             }
         }
     }
 }
 
-::ng-deep .mat-dialog-content {
+::ng-deep #arlas-search-dialog .mat-dialog-content {
```

Comment:
This id will be declared in ARLAS-wui ?

Reply:
No it was already declared into toolkit. When we open the dialog.

Reply:
right ! perfect